source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/Context/ChargesDatabaseContext.cs

```diff
@@ -55,6 +57,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<Charge>().ToTable("Charge");
             modelBuilder.Entity<MarketParticipant>().ToTable("MarketParticipant");
             modelBuilder.Entity<MeteringPoint>().ToTable("MeteringPoint");
+            modelBuilder.Entity<DefaultChargeLink>().ToTable("DefaultChargeLinkSetting");
```

Comment:
After this [PR](https://github.com/Energinet-DataHub/geh-charges/pull/548/files) was merged, please change to this:
```suggestion
            modelBuilder.Entity<DefaultChargeLink>().ToTable("DefaultChargeLink");
```